nistats/utils.py

```diff
@@ -41,6 +42,28 @@ def _check_and_load_tables(tables_, var_name):
     return tables
 
 
+def _verify_events_file_uses_valid_value_separators(events_files):
+    valid_separators = [',', '\t']
+    events_files = [events_files] if isinstance(events_files, str) else events_files
+    for events_file_ in events_files:
+        try:
+            with open(events_file_, 'r') as events_file_obj:
+                events_file_sample = events_file_obj.read(1024)
+        except TypeError as err:
+            pass
+        except IOError:
+            pass
+        else:
+            try:
+                csv.Sniffer().sniff(sample=events_file_sample,
+                                    delimiters=valid_separators,
+                                    )
+            except csv.Error:
+                raise ValueError(
+                    'The values in the events file are not separated by tabs or commas',
```

Comment:
Actually I would put "The values in the events file are not separated by tabs; please enforce BIDS conventions".